benches/intern.rs

```diff
@@ -41,6 +41,7 @@ pub fn intern_and_lookup(c: &mut Criterion) {
                 let mut intern = intern_string::Intern::new();
                 b.iter(|| {
                     for data in dataset {
+                        let data = data.as_str();
```

Comment:
having to do this is annoying. basically, `From<&String>` is not implemented for `Box<str>`, so we do this on our own first. this is... sort of annoying. i am gonna have to think about it. but it's a UI issue, not a performance issue, this is just converting some pointers, not doing a copy or anything.

Reply:
fixed this.